ehsm_kms_service/function.js

```diff
@@ -81,6 +81,57 @@ const _nonce_cache_timer = () => {
     }
 }
 
+const _token_time_verify = (DB) => {
+    setInterval(() => {
+        try {
+            const current_time = new Date()
+                .getTime()
+            const query = {
+                selector: {
+                    //Query criteria
+                    token_expired_time: {
+                        $lt: current_time
+                    },

```

Comment:
using keyidA, could keyidB also be selected?

Reply:
this function will range all keyid  status that finish GetParametersForImport request, and check time.